internal/output/golangci/golangci.yml

```diff
@@ -122,11 +122,18 @@ linters:
     - godox
     - goerr113
     - gomnd
+    - gomoddirectives
     - nestif
     - paralleltest
     - tagliatelle
+    - thelper
     - typecheck
     - wrapcheck
+    # abandoned linters for which golangci shows the warning that the repo is archived by the owner
+    - interfacer
```

Comment:
Do we have a replacement for those?

Reply:
`maligned` ->`fieldalignment`.
`scopelint` -> `exportloopref`.
`golint` -> `revive`.
As per: https://golangci-lint.run/usage/linters/.

For interfacer I don't see any direct replacement unfortunately.